.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Fixes {issue number}
```

Comment:
Does "Fixes" close the related issue?

Reply:
Yeah if typed like `Fixes #5`, is there a better way to reference an issue? Might also change it to “Fixes|Implements”